### ISSUE TITLE
Feedback on content reports

### ISSUE THREAD

**Issue by imax9000** [bluesky-social/atproto#2445]
Add visibility to what happens after reporting a piece of content. 

Users should be able to check on the current status of their reports and what actions have been taken. 

Because right now, from the outside, moderation looks like a black box that does exactly nothing.

**Comment by imax9000** [comment]
Seriously, people, even Twitter is better at this. They at least have the balls to tell you they aren't doing anything about it.

**Comment by bnewbold** [comment]
Yup! We know that this is needed. It is a bit complex with federation and multiple labelers: we need a new communication/messaging channel between arbitrary moderation services and users. Currently our moderation service can send emails, but this doesn't work great (many people don't check email, isn't in-app, email is a separate system, many accounts haven't verified email, etc). And mod services generally can't currently send email like this.

We have a few different ideas, like injecting in-app notifications, or building out a "mod mail" functionality in the PDS itself. This is one of our top mod engineering priorities right now.

**Comment by imax9000** [comment]
Glad to hear that!

You've already got reporting path working, by having PDS talk to Ozone with appropriate credentials. Why not start with expanding that API to allow listing/querying reports along the same path?

**Comment by bnewbold** [comment]
That actually isn't a bad idea, I don't think we'd considered polling the labeler directly. I guess the UI could be another tab on the labeler video page.

**Comment by imax9000** [comment]
I'd vote to put it under "Moderation" - you're already listing subscribed labelers there anyway.

But I'd also be happy to have just the API, and figuring out UI and everything later on.

**Comment by Mennaruuk** [comment]
Do moderators have a section where they classify a report as successfully resolved or otherwise? This could be shared publicly so that when a user clicks on the report ID, they can fetch the status directly from the mods.

**Comment by imax9000** [comment]
> Do moderators have a section where they classify a report as successfully resolved or otherwise? This could be shared publicly so that when a user clicks on the report ID, they can fetch the status directly from the mods.

1) no, it should not be shared publicly, since report IDs are easy to guess (they're a sequentially increasing number)
2) the UI moderators use has it's own separate ACLs, and is not available to just anyone
3) that's not even the easiest way to implement something like this
